test_schemas/en/test_section_summary.json

```diff
@@ -187,6 +187,60 @@
                 }
             ]
         },
+        {
+            "id": "household-count-section",
+            "title": "Household member count",
+            "summary": {
+                "show_on_completion": true,
+                "title": {
+                    "text_plural": {
+                        "forms": {
+                            "one": "{number_of_people} person lives here",
+                            "other": "{number_of_people} people live here"
+                        },
+                        "count": {
+                            "source": "answers",
+                            "identifier": "number-of-people-answer"
+                        }
+                    },
+                    "placeholders": [
+                        {
+                            "placeholder": "number_of_people",
+                            "value": {
+                                "source": "answers",
+                                "identifier": "number-of-people-answer"
+                            }
+                        }
+                    ]
+                }
+            },
+            "groups": [
+                {
+                    "id": "number-of-people-group",
+                    "title": "Number of people",
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "number-of-people",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "number-of-people-answer",
+                                        "label": "How many people live here?",
+                                        "mandatory": true,
+                                        "type": "Number"
+                                    }
+                                ],
+                                "description": "",
+                                "id": "number-of-people-question",
+                                "title": "",
```

Comment:
We should add a title here and remove the description above before my validator PR gets merged 😀

Reply:
Also means I don't need to do a null check on title anymore, which I had to do above

Reply:
Question title is mandatory - https://github.com/ONSdigital/eq-questionnaire-validator/blob/master/schemas/questions/types/general.json#L93

Reply:
Question title is, summary title isn't which is what is used by `title_for_location` method.

Reply:
Ah, context is everything. The comment is against the question title 🤷 .